code/modules/lighting/lighting_object.dm

```diff
@@ -17,6 +17,9 @@
 /atom/movable/lighting_object/Initialize(mapload)
 	. = ..()
 	verbs.Cut()
+	//We avoid setting this in the base as if we do then the parent atom handling will add_atom_color it and that
+	//is totally unsuitable for this object, as we are always changing it's colour manually
```

Comment:
You mean its, not it's (or you can ignore this because NOBODY CARES and I'm a TURBONERD)

Reply:
Trying to tell him off when he makes this mistake but he's as stubborn as a mule